concordium-consensus/src/Concordium/GlobalState/Persistent/Account.hs

```diff
@@ -178,18 +177,16 @@ makeLenses ''PersistentExtraBakerInfo
 
 instance forall av. IsAccountVersion av => Show (PersistentExtraBakerInfo av) where
     show = case accountVersion @av of
-        SAccountV0 -> show
-        SAccountV1 -> show
+        SAccountV0 -> show . _theExtraBakerInfo
+        SAccountV1 -> show . _theExtraBakerInfo
 
-instance forall av m. (IsAccountVersion av, MonadBlobStore m) => Cacheable m (PersistentExtraBakerInfo av) where
-  cache = case accountVersion @av of
-    SAccountV0 -> pure
-    SAccountV1 -> theExtraBakerInfo cache
+instance forall av m. (Applicative m) => Cacheable m (PersistentExtraBakerInfo av)
 
 -- |A persistent version of 'BakerInfoEx'.
-
+-- IMPORTANT NOTE: The 'Cacheable' instance relies on this not requiring recursive caching.
+-- If this changes, the instance for 'BakerInfos' should also be updated.
```

Comment:
Similar to the other comment I have, the cacheable instance of this type is itself trivial. So is the comment talking about some other instances. If so, which ones?